<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Afficher_Patient</class>
 <widget class="QDialog" name="Afficher_Patient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>130</y>
     <width>691</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_R_A">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>       Recherche Avanc√©e</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Recherche">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>16</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>20</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>711</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>tableView</zorder>
  <zorder>Button_R_A</zorder>
  <zorder>label_Recherche</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
